articles/notification-hubs/notification-hubs-app-service.md

```diff
@@ -22,11 +22,13 @@ ms.lastreviewed: 01/04/2019
 
 # Integration with App Service Mobile Apps
 
-> [!NOTE]
-> Microsoft is committed to fully supporting Azure App Service Mobile Apps, including support for the latest OS release, bug fixes, documentation improvements, and community PR reviews. Please note that the product team is not currently investing in any new feature work for Azure Mobile Apps. We highly appreciate community contributions to all areas of Azure Mobile Apps.
-
 To facilitate a seamless and unifying experience across Azure services, [App Service Mobile Apps](../app-service-mobile/app-service-mobile-value-prop.md) has built-in support for push notifications using Notification Hubs. [App Service Mobile Apps](../app-service-mobile/app-service-mobile-value-prop.md) offers a highly scalable, globally available mobile application development platform for Enterprise Developers and System Integrators that brings a rich set of capabilities to mobile developers.
 
+Visual Studio App Center delivers a more modern mobile solution for developers today. It supports end to end and integrated services central to mobile app development. Developers can use **Build**, **Test** and **Distribute** services to set up Continuous Integration and Delivery pipeline. Once the app is deployed, developers can monitor the status and usage of their app using the **Analytics** and **Diagnostics** services, and engage with users using the **Push** service. Developers can also leverage **Auth** to authenticate their users and **Data** service to persist and sync app data in the cloud.
```

Comment:
Please don't capitalize Continuous Integration and Delivery unless they are proper names of Azure services.